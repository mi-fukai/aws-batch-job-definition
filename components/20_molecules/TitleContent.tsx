import React from "react";
import Link from "next/link";

import styles from "@/styles/20_molecules/TitleContent.module.css";

export const TitleContent: React.VFC = () => {
  return (
    <div className={styles.content}>
      <Link href="/">
        <a>
          <h1 className={styles.title}>Elasticてすと</h1>
        </a>
      </Link>
      <p>
        きーわーどを入力して検索すると、Feedの全要素から一致する文字列があるか検索します
      </p>
    </div>
  );
};

export default TitleContent;
