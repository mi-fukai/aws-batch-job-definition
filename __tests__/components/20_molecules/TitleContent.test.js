import React from "react";
import { cleanup, render } from "@testing-library/react";
import TitleContent from "@/components/20_molecules/TitleContent";

afterEach(cleanup);

describe("Snap Shot test", () => {
  it("Component Global Snap Shot", () => {
    const { asFragment } = render(<TitleContent />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
