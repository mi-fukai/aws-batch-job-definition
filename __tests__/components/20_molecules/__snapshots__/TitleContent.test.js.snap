// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snap Shot test Component Global Snap Shot 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    <a
      href="/"
    >
      <h1
        class="title"
      >
        Elasticてすと
      </h1>
    </a>
    <p>
      きーわーどを入力して検索すると、Feedの全要素から一致する文字列があるか検索します
    </p>
  </div>
</DocumentFragment>
`;
